src/Validator/Constraints/InlineConstraint.php

```diff
@@ -51,6 +51,9 @@ public function __construct($options = null)
 
     public function __sleep(): array
     {
+        // Initialize "groups" option if it is not set
+        $this->groups;
```

Comment:
Maybe we should add `@see parent::__sleep()` ?

Reply:
In fact, I added this but it's look like it cause a `pslam`error